optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -351,6 +357,7 @@ def __init__(
         best_params=None,  # type: Optional[Dict[str, Any]]
         tuning_history=None,  # type: Optional[List[Dict[str, Any]]]
         study=None,  # type: Optional[Study]
+        optuna_callbacks=None,  # type: Optional[List[Callable[[Study, FrozenTrial], None]]]
```

Comment:
Could you add a test case to check if a given callback is actually called?

Reply:
Thank you for your review. I added a test at dd790e6.